utils/firestore.js

```diff
@@ -764,33 +763,43 @@ const updateBox = async (id, data, loginSite) => {
 }
 
 
-const removeBag = async (institute, requestData) => {
+const removeBag = async (siteCode, requestData) => {
     let boxId = requestData.boxId;
     let bags = requestData.bags;
     let currDate = requestData.date;    
-    const snapshot = await db.collection('boxes').where('132929440', '==', boxId).where('siteAcronym', '==',institute).get();
+    const snapshot = await db.collection('boxes').where('132929440', '==', boxId).where('789843387', '==',siteCode).get();
     if(snapshot.size === 1){
-        let box = snapshot.docs[0];
-        let data = box.data()
-        let currBags = data.bags;
-        let bagIds = Object.keys(currBags);
-        for(let i = 0; i < bags.length; i++){
-            if(currBags.hasOwnProperty(bags[i])){
-                delete currBags[bags[i]]
+        let doc = snapshot.docs[0];
+        let box = doc.data();
+        let bagConceptIDs = ["147157381", "147157382", "147157383", "147157384", "147157385", "147157386", "147157387", "147157388", "147157389", "147157390", "147157391", "147157392", "147157393", "147157394", "147157395"];
```

Comment:
Thoughts on putting this data structure into a shared function similar to what you did with the site codes?

Reply:
We can move it to shared.js file.

Reply:
Perfect, thanks.